bin/prettier.js

```diff
@@ -9,25 +9,8 @@ const minimist = require("minimist");
 const prettier = require("../index");
 
 const argv = minimist(process.argv.slice(2), {
-  boolean: [
-    "write",
-    "stdin",
-    "single-quote",
-    "trailing-comma",
-    "bracket-spacing",
-    // The supports-color package (a sub sub dependency) looks directly at
-    // `process.argv` for `--no-color` and such-like options. The reason it is
-    // listed here is to avoid "Ignored unknown option: --no-color" warnings.
-    // See https://github.com/chalk/supports-color/#info for more information.
-    "color",
-    "help",
-    "version",
-    "debug-print-doc",
-    "debug-check",
-    // Deprecated in 0.0.10
-    "flow-parser"
-  ],
-  string: ["print-width", "tab-width", "parser"],
+  boolean: true,
```

Comment:
I don't know what the right answer is. I had to do this because it's the only way to tell minimist to parse `--flag` options as boolean by default and use equals to specify parameters (`--flag=param`). Otherwise, I can't have both `--trailing-comma` and `--trailing-comma=all`, because it also parses `--trailing-comma all`, which doesn't work when doing `prettier --trailing-comma foo.js`. It assigns `foo.js` to the valie of the `trailing-comma` option.

I'm not loving minimist. Is there another argument parser that we could use?